app/renderer/components/preferences/syncTab.js

```diff
@@ -82,12 +83,47 @@ class SyncTab extends ImmutableComponent {
           </div>
         </div>
       </div>
+      {this.enabled ? this.devicesContent : null}
       <SettingItem>
-        <Button l10nId='syncNewDevice' className='whiteButton' onClick={this.props.showOverlay.bind(this, 'syncNewDevice')} />
+        <Button l10nId='syncNewDevice' className='primaryButton' onClick={this.props.showOverlay.bind(this, 'syncNewDevice')} />
       </SettingItem>
     </SettingsList>
   }
 
+  get devices () {
+    let devices = []
+    this.props.syncData.get('devices').forEach((device, id) => {
+      devices.push({id, name: device.name, lastFetchTimestamp: device.lastFetchTimestamp})
+    })
+    return devices
+  }
+
+  get devicesContent () {
+    return <div className='devices'>
+      <div className='sectionTitle' data-l10n-id='syncDevices' data-test-id='syncDevices' />
+      <SortableTable
+        headings={['', 'syncDeviceName', 'syncDeviceLastActive']}
+        defaultHeading='syncDeviceLastActive'
+        defaultHeadingSortOrder='desc'
+        rows={this.props.syncData.get('devices').map((device, id) => [
```

Comment:
i think this needs to check that `this.props.syncData.get('devices')` is not null/undefined, may be the cause of the error i saw